pkg/apis/core/v1/core.proto

```diff
@@ -116,4 +117,19 @@ message Health {
 message HealthStatus {
   Health health = 1;
   Status status = 2;
+}
+
+message AlertLog {
+  uint64 timestamp = 1; // unix epoch timestamp
+  repeated string labels = 2;
+  google.protobuf.Struct metadata  = 3;
+}
+
+message AlertLogList {
+  repeated AlertLog alertEvents = 1;
```

Comment:
For consistency, name this field `items`

Reply:
Done